miasm2/arch/mep/sem.py

```diff
@@ -0,0 +1,1172 @@
+# Toshiba MeP-c4 - miasm instructions side effects
+# Guillaume Valadon <guillaume@valadon.net>
+
+from miasm2.core.sembuilder import SemBuilder
+from miasm2.ir.ir import IntermediateRepresentation
+from miasm2.arch.mep.arch import mn_mep
+from miasm2.arch.mep.regs import PC, SP, LP, SAR, TP, RPB, RPE, RPC, EPC, NPC, \
+    take_jmp, in_erepeat
+from miasm2.arch.mep.regs import EXC, HI, LO, PSW, DEPC, DBG
+from miasm2.expression.expression import ExprId, ExprInt, ExprOp
+from miasm2.expression.expression import ExprAff, ExprCond, ExprMem
+from miasm2.core.cpu import sign_ext
+
+from miasm2.arch.mep.regs import exception_flags
+MEP_EXCEPTION_ZERO_DIVIDE = 1 << 0
```

Comment:
Can you use:
```python
from miasm2.jitter.csts import EXCEPT_DIV_BY_ZERO
```

Reply:
Is there a way to make the constant seen by the sembuilder ?

I did no manage to do it, that's why I am using my own, see
https://github.com/cea-sec/miasm/pull/763/files#diff-199ffae7a66bdc2eeb8cc43346463088R941